src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,33 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.dao.MovieDao;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.dao.impl.CinemaHallDaoImpl;
+import mate.academy.dao.impl.MovieDaoImpl;
+import mate.academy.dao.impl.MovieSessionDaoImpl;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.impl.CinemaHallServiceImpl;
+import mate.academy.service.impl.MovieServiceImpl;
+import mate.academy.service.impl.MovieSessionServiceImpl;
 
 public class Main {
     public static void main(String[] args) {
```

Comment:
Let's stick to the task description 

In the mate/academy/Main.main() method create an instance of AuthenticationService using injector and test all methods from it.